### PR TITLE
Made a sentence slightly more english

### PR DESCRIPTION
**Pull request by GTheron**
Replaced "Even more" with "What's more" to make a sentence "more english".

### CODE REVIEW

**Comment by zatikbalazs** [comment]
It sounds better indeed.

**Comment by javiereguiluz** [comment]
:+1: It looks like an easy win. Thanks @GTheron

File: components/http_foundation/introduction.rst [hunk 1]
```diff
@@ -464,7 +464,7 @@ represented by a PHP callable instead of a string::
     you must call ``ob_flush()`` before ``flush()``.
 
     Additionally, PHP isn't the only layer that can buffer output. Your web
-    server might also buffer based on its configuration. Even more, if you
+    server might also buffer based on its configuration. What's more, if you
     use fastcgi, buffering can't be disabled at all.
```
Comment (xabbuh) [inline components/http_foundation/introduction.rst]: I think we can also change "fastcgi" to "FastCGI".

**Comment by xabbuh** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you Gabriel.

**Comment by xabbuh** [comment]
@GTheron I merged your change into the `2.3` branch (that's why it is shown as closed) and made the tweak to "fastcgi" in 6b5de8c.

**Comment by GTheron** [comment]
Brilliant, thanks

**Pull request closed without merging**